### PR TITLE
Feature/refactoring news

### PR DESCRIPTION
**Pull request by arejula27**

### CODE REVIEW

File: src/database/model/news/news.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    )
+    alternative_headline: str | None = Field(
+        max_length=500, default=None, schema_extra={"example": "Example news alternative headline"}
+    )
+
+
+class News(NewsBase, table=True):  # type: ignore [call-arg]
+    __tablename__ = "news"
+    identifier: int = Field(primary_key=True, foreign_key="ai_asset.identifier")
+    news_categories: List[NewsCategory] = Relationship(
```
Comment (josvandervelde) [inline src/database/model/news/news.py:40]: In previous implementation there were also business categories. Are they omitted on purpose?

File: src/database/model/news/news_category.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from sqlmodel import SQLModel, Field
+
+
+class NewsCategoryNewsLink(SQLModel, table=True):  # type: ignore [call-arg]
```
Comment (josvandervelde) [inline src/database/model/news/news_category.py:4]: Rename this file to `news_category_link.py`?

**Comment by josvandervelde** [comment]
Could you also remove the `todo/news.py` file?

File: src/database/model/news/news.py (hunk 1 above)
Comment (arejula27) [reply src/database/model/news/news.py:40]: They are implemented on the pr #52, i ca waint until it is merged or i can implement the model/general/business_category copying the cocntent

Comment (josvandervelde) [reply src/database/model/news/news.py:40]: A sorry you already said that in the chat :) Let's merge this and fix it in #52

**Comment by arejula27** [comment]
> Could you also remove the `todo/news.py` file?

Done

**Pull request merged**